pinc/misc.inc

```diff
@@ -444,6 +444,27 @@ function javascript_safe($str, $encoding=NULL)
     return $str;
 }
 
+// -----------------------------------------------------------------------------
+
+function search_safe($str)
```

Comment:
I can imagine this function being used in non-search contexts, so it probably shouldn't be called `search_safe`. How about `normalize_whitespace`?

Reply:
Renamed in d32dca8c4f0ea152e3a4b5e9d600d49b5bf78723